Makefile-libs.Inc

```diff
@@ -75,10 +75,10 @@ $(BINDIR_POSTRS_SETUP_LIBS): $(PROJ_DIR)$(POSTRS_SETUP_ZIP)
 	touch $@
 
 $(PROJ_DIR)$(POSTRS_SETUP_ZIP):
-	curl -sSL --retry 5 --retry-max-time 60 $(POSTRS_SETUP_URL_ZIP) -o $(PROJ_DIR)$(POSTRS_SETUP_ZIP)
+	wget -nv -c $(POSTRS_SETUP_URL_ZIP) -O $(PROJ_DIR)$(POSTRS_SETUP_ZIP)
```

Comment:
It's not an issue with `curl`. It rather seems like some github servers actively reset connections from our build agent to them. Without a proxy I doubt we can solve the issue.

It seems the issue persists, then we will have to move our self-hosted build agent from `unittests` to `unittests-slower`. Those are allowed to fail and don't block us from merging pull requests. Just removing `[self-hosted, linux, arm64]` from `unittests:strategy:matrix:os` and adding it to `unittests-slower:strategy:matrix:os` should suffice to work around the issue for now until we find a more permanent solution.

Reply:
This was an attempt to get retries happening since `--retry-all-errors` didn't work on all platforms for curl, wget retries 20 times by default. But it does not seem to be making a difference.